src/utils.ts

```diff
@@ -0,0 +1,235 @@
+export const get_from_dict = (obj: object, key: string): any => {
+    let keyParts: string[] = String(key).split('.');
+    let current: any = obj;
+
+    for (let i = 0; i < keyParts.length; i++) {
+        // @ts-ignore
+        const part: string = keyParts[i];
```

Comment:
`: string` not needed, `// @ts-ignore` not needed.

Reply:
Changed from 

```
for (let i = 0; i < keyParts.length; i++) {
        // @ts-ignore
        const part: string = keyParts[i];
```

to

`for (let part of keyParts) {`

Reply:
Oh yeah, that's even better. Also personally I prefer using `const` rather than `let` where possible because it's a little clearer about what the variable is for and sometimes it matters for TypeScript too, but I don't care enough to make you do it everywhere :)